src/domains/Work/components/WorkEdit/presentations/items/InfoPopOver.tsx

```diff
@@ -0,0 +1,27 @@
+import type { FC, ReactNode } from 'react';
+
+import { Info } from 'lucide-react';
+
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@/components/ui/Tooltip';
+
+type Props = {
+  children: ReactNode;
+};
+
+export const InfoPopOver: FC<Props> = ({ children }) => (
+  <TooltipProvider>
+    <Tooltip>
+      <TooltipTrigger>
+        <Info size={16} color="#757575" />
+      </TooltipTrigger>
+      <TooltipContent className="rounded-md relative text-xl border-2 bg-white border-orange-pop">
+        <p className="whitespace-pre-wrap text-[10px] font-bold">{children}</p>
+      </TooltipContent>
```

Comment:
```suggestion
      <TooltipContent className="rounded-md relative text-xl border-2 bg-white border-orange-pop">
      {
          supportedExts.map(({category,exts})=>(
              <Vertical>
                 <Typography>
                     {category}
                 </Typography>
                 <span>
                 [
                 <Vertical>
                 {
                    exts.map((ext)=>(
                    <p>
                        .{ext}
                    </p>
                    ))
                 }
                 <Vertical>
                 ]
              </Vertical>
          ))
      }
      </TooltipContent>
```


冗長ではあると思うけど私だったらこういうふうに書くかも(動くかどうかは確かめてない、あくまでイメージ)

supportedExtっていうpropsを取ってそれを元にPopoverがレンダリングする

supportedExtは`category`と`exts`をkeyに持つ構造体の配列

で親からsupportedExtを渡すだけでpopoverが統一されたスタイルでレンダリングをしてくれる

親が渡すsupportedExtは固定値で持っておけば何か変更があったとしても構造体を変えるだけでそれまでと同様のスタイルが保てる(文字列だと開発者がスタイルを気にして開発しなきゃいけなくなる(ドットだったりスペースだったり))

Reply:
ありがとうございます 勉強になりま
そんな感じにしてみます